plugins/GSdx/GSDeviceOGL.cpp

```diff
@@ -247,7 +247,7 @@ bool GSDeviceOGL::Create(GSWnd* wnd)
 	m_convert.cb = new GSUniformBufferOGL(g_convert_index, sizeof(ConvertConstantBuffer));
 	// Upload once and forget about it
 	ConvertConstantBuffer cb;
-	cb.ScalingFactor = GSVector4i(theApp.GetConfig("nativeres", 0) ? 1 : theApp.GetConfig("upscale_multiplier", 2));
+	cb.ScalingFactor = GSVector4i(theApp.GetConfig("upscale_multiplier", 1)==1 ? 1 : theApp.GetConfig("upscale_multiplier", 2));
```

Comment:
It would mean that when a custom scaling factor is used `cb.ScalingFactor = 1`, which is not true in the above code.